src/components/TheHeader.vue

```diff
@@ -17,10 +17,10 @@
       <v-toolbar-items v-if="!mobile" class="hidden-xs-only">
         <v-tabs class="pa-2">
           <v-tab
-            v-for="({ text, icon, page }, i) in pageLinks"
+            v-for="({ text, icon, page, cy_name }, i) in pageLinks"
```

Comment:
Let's use `camelCase` for variable and attribute names instead
```suggestion
            v-for="({ text, icon, page, cyName }, i) in pageLinks"
```